crates/bin/pd/src/main.rs

```diff
@@ -166,14 +166,12 @@ async fn main() -> anyhow::Result<()> {
             };
 
             // Configure a Prometheus recorder and exporter.
+            use penumbra_dex::component::metrics::PrometheusBuilderExt;
             let (recorder, exporter) = PrometheusBuilder::new()
                 .with_http_listener(metrics_bind)
                 // Set explicit buckets so that Prometheus endpoint emits true histograms, rather
                 // than the default distribution type summaries, for time-series data.
-                .set_buckets_for_metric(
-                    metrics_exporter_prometheus::Matcher::Prefix("penumbra_dex_".to_string()),
-                    penumbra_dex::component::metrics::DEX_BUCKETS,
-                )?
+                .set_buckets_for_dex_metrics()?
```

Comment:
see the relevant commit for more info, but this (a) cuts down on non-local reasoning, and (b) sets the stage for us to add counters and gauges to this component.